// PlayerNetPackOpcodes.h

HandleOpcode(CMSG_ACTIVITY_ASK);
HandleOpcode(CMSG_COUNTRY_TASK_INVEST);
HandleOpcode(CMSG_COUNTRY_INVEST_PLAYER);
HandleOpcode(CMSG_COUNTRY_TASK_INVEST_TASK_REWARD);
HandleOpcode(CMSG_COUNTRY_INVEST_DONATE);
HandleOpcode(CMSG_COUNTRY_TAST_OCCUPY);
HandleOpcode(CMSG_COUNTRY_OCCUPY_PLAYER);
HandleOpcode(CMSG_COUNTRY_OCCUPY_TAKE_REWARD);
HandleOpcode(CMSG_PERSONAL_TASK);
HandleOpcode(CMSG_PERSONAL_TASK_ONE);
HandleOpcode(CMSG_PERSONAL_TAKE_AWARD);
HandleOpcode(CMSG_COUNTRY_TASK_INVEST_CLEAR_CD);
HandleOpcode(CMSG_COUNTRY_TASK_DEFENCEBORDERLAND);
HandleOpcode(CMSG_COUNTRY_DEFENCEBORDERLAND_PLAYER);
HandleOpcode(CMSG_COUNTRY_DEFENCEBORDERLAND_TAKE_REWARD);
HandleOpcode(CMSG_DEFENCEBORDERLAND_CALL_UP);

